pxf/pxf-hive/src/main/java/org/apache/hawq/pxf/plugins/hive/HiveMetadataFetcher.java

```diff
@@ -50,9 +50,21 @@ public HiveMetadataFetcher(InputData md) {
         client = HiveUtilities.initHiveClient();
     }
 
+    /**
+     * Fetches metadata of hive tables corresponding to the given pattern
+     * For patterns matching more than one table, the tables are skipped.
```

Comment:
Maybe For patterns matching more than one table, the unsupported tables are skipped.?